libs/src/api/user.js

```diff
@@ -61,9 +61,14 @@ class Users extends Base {
    * @example
    * getFollowIntersectionUsers(100, 0, 1, 1) - IDs must be valid
    */
-  async getFollowIntersectionUsers (limit = 100, offset = 0, followeeUserId, followerUserId) {
-    this.REQUIRES(Services.DISCOVERY_PROVIDER)
-    return this.discoveryProvider.getFollowIntersectionUsers(limit, offset, followeeUserId, followerUserId)
+  async getFollowIntersectionUsers (limit = 100, offset = 0, followeeUserId) {
```

Comment:
let's rename this method to

getMutualFollowers

the method on discprov service wrapper can stay the same, but i always have trouble parsing this one :-)